mozjs/src/conversions.rs

```diff
@@ -525,6 +525,15 @@ impl FromJSValConvertible for f64 {
     }
 }
 
+/// Copies chars to the string
+unsafe fn fast_copy(chars: &[u8]) -> String {
```

Comment:
Can we inline this function, or do you still need this function for the SIMD follow-up?
If we want to keep this function I would suggest:
- Adding safety doc
- renaming chars to latin1_bytes or latin1_chars to avoid confusion with other possible char types.
- use unsafe block around unsafe functions (something we should be doing more of in mozjs anyway)

Reply:
Removed the function. The simd stuff doesn't give us any improvements. Probably because encoding_rs does already anything it can.